cmd/commands.go

```diff
@@ -21,6 +26,75 @@ var Search = &cobra.Command{
 	},
 }
 
+var (
+	remoteName    string
+	repoDirectory string
+)
+
+var ByRemote = &cobra.Command{
+	Use:   "by-remote",
+	Short: "Search for github profile by remote URL",
+	Run: func(cmd *cobra.Command, args []string) {
+		gitArgs := []string{
+			"-C",
+			repoDirectory,
+			"remote",
+			"get-url",
+			remoteName,
+		}
+		gitCmd := exec.Command("git", gitArgs...)
+		remoteBytes, err := gitCmd.Output()
+
+		if err != nil {
+			log.Fatalf("failed to get remote URL: %v", err)
+		}
+
+		owner, err := help.OwnerFromRemote(string(remoteBytes))
+
+		if err != nil {
+			log.Fatalf("failed to get owner from remote URL: %v", err)
+		}
+
+		results := gh.SearchUser(tui.Ctx, tui.RestClient, owner)
+		total := results.GetTotal()
+
+		if total == 0 {
+			log.Fatalf("No user/organization found matching %q", owner)
+		} else if total > 1 {
+			tui.RenderList(results)
+		} else {
```

Comment:
If there is more than 1 result, the list to select the search results is rendered. But, since the owner in the remote URL *should* be an exact match to the username that the user wants to display, I was wondering if that username should be automatically selected.